corehq/apps/domainsync/management/commands/copy_group_data.py

```diff
@@ -62,25 +63,12 @@ def handle(self, *args, **options):
         if not exclude_user_owned:
             owners.extend(group.users)
 
-        def keys_for_owner(domain, owner_id):
-            return [
-                [domain, owner_id, False],
-                [domain, owner_id, True],
-            ]
-
-        def get_case_ids(owners):
-            keys = list(itertools.chain(*[keys_for_owner(domain.name, owner_id) for owner_id in owners]))
-            results = sourcedb.view(
-                'hqcase/by_owner',
-                keys=keys,
-                reduce=False,
-                include_docs=False,
-            )
-            return [res['id'] for res in results]
-
         print 'getting case ids'
 
-        case_ids = get_case_ids(owners)
+        with OverrideDB(CommCareCase, sourcedb):
```

Comment:
It maybe could, though I think we'd want something that does it a bit less clunkily—one could imagine how annoying it would look to nest every single db call under `with OverrideDB(CommCareCase, get_customer_database(CommCareCase, domain)):`!